src/main/kotlin/blackjack/domain/Person.kt

```diff
@@ -0,0 +1,12 @@
+package blackjack.domain
+
+class Person(val name: String, val ownCards: OwnCards = OwnCards(), private var state: State = State.CONTINUE) {
+    fun addCard(drawYn: Boolean) {
+        ownCards.addCard(drawYn)
+
+        if (ownCards.sumCardNumber() > Draw.DRAW_LIMIT) state = State.LOSE
+        else if (ownCards.sumCardNumber() == Draw.DRAW_LIMIT) state = State.WIN
+    }
+
+    fun checkDrawable(drawYn: Boolean): Boolean = state == State.CONTINUE && drawYn
```

Comment:
```suggestion
    fun checkDrawable(drawYn: Boolean): Boolean = state == (State.CONTINUE && drawYn)
```

비교에 대해서는 ()로 감싸주는게 좋다고 생각해요~